<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrateAxesDialog</class>
 <widget class="QDialog" name="CalibrateAxesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gimbal Axis Calibration Progress</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="pitchStatus_label">
     <property name="text">
      <string>Pitch Status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="rollStatus">
     <property name="text">
      <string>Waiting to Calibrate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QProgressBar" name="rollProgress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="rollStatus_label">
     <property name="text">
      <string>Roll Status:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="yawStatus">
     <property name="text">
      <string>Waiting to Calibrate</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QProgressBar" name="yawProgress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="yawStatus_label">
     <property name="text">
      <string>Yaw Status:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QProgressBar" name="pitchProgress">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="pitchStatus">
     <property name="text">
      <string>Waiting to Calibrate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QFrame" name="cancelButtonFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
